proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -766,8 +790,8 @@ ProToken id(int inputTokenType) throws IOException {
         case '\'':
           append();
           getChar();
-          if (ttype == ProParserTokenTypes.ID)
-            ttype = ProParserTokenTypes.FILENAME;
+          if (ttype == PreprocessorParser.ID)
+            ttype = PreprocessorParser.FILENAME;
           break;
         default:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1151)